src/main/groovy/org/pih/warehouse/importer/DataImporter.groovy

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse.importer
+
+interface DataImporter {
+
+    List<Map> getData()
+
+    void validateData(ImportDataCommand command)
+
+    void importData(ImportDataCommand command)
+}
```

Comment:
since the `ImportDataService` import is almost the same as this one, maybe let's just extend this one and add the `getData()` method here?
```groovy
interface DataImporter extends ImportDataService {
  List<Map> getData()
}
```

Reply:
This seems awkward but it's too late for me to think clearly about a solution so let's discuss during tomorrow's tech huddle.